### PR TITLE
Added normalized initialization.

### PR DESCRIPTION
**Pull request by Thrandis**
Got a bug on my own computer, so I'm checking if the problem is on my side.

### CODE REVIEW

**Comment by Thrandis** [comment]
@dmitriy-serdyuk do you want something else for this ccw?

File: blocks/extras/initialization.py [hunk 1]
```diff
@@ -34,3 +34,19 @@ def make_matrix(size, perm):
             return make_matrix(shape[0], self.permutation)
         else:
             return make_matrix(shape[0], rng.permutation(shape[0]))
+
+
+class NormalizedInitialization(NdarrayInitialization):
+    u"""Initialize the parameters with the Glorot method, as introduced in
```
Comment (dmitriy-serdyuk) [inline blocks/extras/initialization.py]: There is no scrutinizer in this repo, but the first line should be a short description. Your description doesn't fit in one line, so you'll have to add a bit more detailed paragraph after a blank line.

Comment (dwf) [inline blocks/extras/initialization.py]: We should probably get Scrutinizer set up on this repo even if we don't
have Travis.

**Comment by Thrandis** [comment]
@dmitriy-serdyuk like this?

**Comment by dmitriy-serdyuk** [comment]
Yes, looks nice. Can you add, that it can be applied only to 2D weights and won't work with the convolution, for example?

**Comment by dwf** [comment]
Also, IIRC this is meant for a particular type of activation function, and there is a separate derivation to be made for others?

**Comment by Thrandis** [comment]
@dmitriy-serdyuk all right!
@dwf ahhh yeah you're right as well! I think it was for tanh, but I will re-read the paper quickly!

**Comment by Thrandis** [comment]
@dwf For ReLUs, there is the derivation for the convolution layers in this paper: http://arxiv.org/pdf/1502.01852v1.pdf
I don't know how much it is used in parctice! What do you think?

**Comment by dwf** [comment]
It's very new, but it'd be nice to have -- no need to do it in this pull request though.

I think there's a derivation floating around for logistic units as well.

**Comment by Thrandis** [comment]
@dmitriy-serdyuk I think #14 must be merged before mine, in order to fix the Scrutinizer.

**Comment by dwf** [comment]
It's in!

**Comment by Thrandis** [comment]
@dwf great, I'll update my code then!

**Pull request closed without merging**